test/lib/usd/translators/UsdExportRootTest/test_usdExport.py

```diff
@@ -0,0 +1,49 @@
+from pymel.core import *
+import maya.cmds as cmds
+import tempfile
+import os
+
+def main():
+    tmpdir = tempfile.gettempdir()
+    scn_dir = os.path.dirname(os.path.realpath(__file__))
+    
+    cmds.loadPlugin('mayaUsdPlugin')
+    cmds.loadPlugin('pxrUsd')
+    cmds.file('{}/UsdExportRootTest.ma'.format(scn_dir), type="mayaAscii", o=True)
+   
+    select(['pCone1', 'Cube1'], r=True)
+    cmds.usdExport(file='{}/only_set1_to_top.usda'.format(tmpdir), sl=True, root="Top")
```

Comment:
Can you please open the stages afterwards and make sure the paths are as expected?